src/tests/ftest/osa/osa_offline_parallel_test.py

```diff
@@ -252,10 +177,5 @@ def test_osa_offline_parallel_test(self):
 
         :avocado: tags=all,pr,hw,large,osa,osa_parallel,offline_parallel
         """
-        # Perform drain testing with 1 to 2 pools
-        # Two pool testing blocked by DAOS-5333.
-        # Fix range from 1,2 to 1,3
-        self.run_offline_parallel_test(1)
-        # Perform drain testing : inserting data in pool
-        # Bug : DAOS-4946 blocks the following test case.
-        # self.run_offline_parallel_test(1, True)
+        # Run the parallel offline test.
+        self.run_offline_parallel_test(1, True)
```

Comment:
Do we need to try the range from 1 to 3 like we did in osa_offline_drain.py?

Reply:
Presently, I don't want to try it... I may need some fine tuning in the script. It causes issue now. Will try it later.